extensions/kafka-client/runtime/src/main/java/io/quarkus/kafka/client/runtime/KafkaRecorder.java

```diff
@@ -71,4 +74,16 @@ private static File extractLibraryFile(URL library, String name) {
         return extractedLibFile;
     }
 
+    public void checkBoostrapServers() {
+        Config config = ConfigProvider.getConfig();
+        Boolean serviceBindingEnabled = config.getValue("quarkus.kubernetes-service-binding.enabled", Boolean.class);
+        if (!serviceBindingEnabled) {
+            return;
+        }
+        Optional<String> boostrapServersOptional = config.getOptionalValue("kafka.bootstrap.servers", String.class);
```

Comment:
Hum... that won't work when using multiple brokers. But let's forget this for now.

Reply:
Good point! Something to keep in mind for the future

Reply:
Good point! Something to keep in mind for the future